### ISSUE TITLE
Slowly scrolling tabs bar with trackpad jittering

### ISSUE THREAD

**Issue by brandonmcconnell** [microsoft/vscode#188160]
Screen recording:

https://github.com/microsoft/vscode/assets/5913254/847a6c30-3a70-44dc-93cb-ef0c5bb00216

This occurs when using either the external or built-in trackpad. There are no other input devices connected and in use when this occurs.

**Comment by bpasero** [comment]
This is not a regression and seems to reproduce for a very long time. If someone has a clue and wants to investigate, we apply the scroll state here to the tabs:

https://github.com/microsoft/vscode/blob/76415ef0b1f60e0479bdfee173c1a4f97e785b52/src/vs/workbench/browser/parts/editor/tabsTitleControl.ts#L217-L221

I feel the issue comes from our custom scrollable implementation, but cannot pin point it yet.

**Comment by bpasero** [comment]
I went deep into debugging this and feel this case cannot be fixed but maybe improved (in https://github.com/microsoft/vscode/pull/189034). The tabs control scrollbar is configured in a way that both scrolling vertically and horizontally is allowed. When using the trackpad I see myself often getting into a state where I end up scrolling both vertically and horizontally when scrolling slowly.  

We do have logic to prefer the scroll direction with larger value, but in some cases the values are identical and as such compete. Today we pick one of the values but I think for the tabs stripe it would be better to ignore the scroll in such case.

This would not fix stutter when slowly scrolling but I see it preventing weird direction turns of the stripe when scrolling slowly.

**tl;dr;** idea would be to ignore scroll events for scrollbars that map `y` to `x` where we see competing directions that would equal each other out.
